example/example.py

```diff
@@ -45,6 +45,14 @@
     account_id=next(iter(account_info)), # Replace with your account number
     dry_run=True # If dry_run=True, we won't place the order, we'll just verify it.
 )
+# Get the lot info for a position in an account
```

Comment:
Nit, can we move this code block so that it's not in between an dry run order being placed (on line 41) and the result of the dry run being printed (line 57)?